### DISCUSSION TITLE
Support for ST_AsMVT?

### DISCUSSION THREAD

**Discussion by donnyv** [duckdb/duckdb-spatial#392] [Ideas]
Is there possible support in the future for ST_AsMVT? Would be cool to be able to feed in WKT or Geojson and output MVT Tiles. Could then use duckdb as part of a dynamic MVT Tile server with different datasources.

**Comment by Maxxen** [comment]
Not yet, but it's planned. Now that we have spatial indexes it actually becomes somewhat feasible to use duckdb as a tile backend as filtering on bounding boxes don't have to perform a full table scan. See https://github.com/duckdb/duckdb_spatial/issues/241
